pkg/controllers/managementuser/clusterauthtoken/register.go

```diff
@@ -93,9 +93,9 @@ func registerDeferred(ctx context.Context, cluster *config.UserContext, extAPICo
 	clusterAuthTokenLister := cluster.Cluster.ClusterAuthTokens(namespace).Controller().Lister()
 	clusterUserAttribute := cluster.Cluster.ClusterUserAttributes(namespace)
 	clusterUserAttributeLister := cluster.Cluster.ClusterUserAttributes(namespace).Controller().Lister()
-	clusterConfigMap := cluster.Core.ConfigMaps(namespace)
-	clusterConfigMapLister := cluster.Core.ConfigMaps(namespace).Controller().Lister()
-	clusterSecret := cluster.Core.Secrets(namespace)
+	clusterConfigMap := cluster.Corew.ConfigMap()
+	clusterConfigMapLister := cluster.Corew.ConfigMap().Cache()
+	clusterSecret := cluster.Corew.Secret()
```

Comment:
Before this change, the controller/lister only had access to resources within the specified namespace, but it looks like that scope is now broader. Can you clarify if this behavior is intentional?

Reply:
TL;DR: Wrangler controllers do not allow creating a namespaced-client.

BTW this is not completely true:
> the controller/lister only had access to resources within the specified namespace, but it looks like that scope is now broader

Note that besides `Get/List/Delete` methods, their `Namespaced` version also existed, so access was not actually restricted. It was only convenient for omitting the namespace in every call. I've adjusted all those calls to explicitly specify the namespace, as it's now a required argument.